qutip/solver/heom/bofin_baths.py

```diff
@@ -8,15 +8,22 @@
 https://github.com/tehruhn/bofin) which was itself derived from an earlier
 implementation in QuTiP itself.
 """
-
+from time import time
 import enum
-
 import numpy as np
-from scipy.linalg import eigvalsh
 
+try:
+    from mpmath import mp
+except ModuleNotFoundError:
+    print(
+        'The mpmath module is needed for the description of Ohmic baths')
+
```

Comment:
I agree that's a better fix, when I did it I was just trying to get around the tests, didn't give it too much thought. It's great to know people in scipy are working on it, i'll keep an eye on it so that I know when it's released ,  I wasn't sure how to answer these, so I ended up doing a commit that implements most suggestions including this one.